code/script.js

```diff
@@ -222,96 +226,140 @@ const generateBoard = () => {
   })
 }
 
+
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
+
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
+  
   // What else should happen when we start the game?
+  charactersInPlay = CHARACTERS
+  generateBoard()
+  setSecret()
+  board.style.display = 'flex'
+  winOrLose.style.display = 'none'
 }
 
+start()
+
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
-
+  const value = questions.options[questions.selectedIndex].value
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
-  // const value =
+   const value1 = questions.value
 
   currentQuestion = {
     category: category,
-    // value: value
+    value: value
```

Comment:
Something I learned about objects recently is when the key equals the value, there is a shorthand way of writing it like this:

`currentQuestion = {
    category,
    value
}`

If you don't add an : sign inside a {}, the object automatically knows that the key is the same as the value. Future shortcut!